lib/tools/test/cprof_SUITE.erl

```diff
@@ -215,12 +215,9 @@ on_load_test(Config) ->
     M4__4 = M*4 - 4,
```

Comment:
Nitpick; `M_1` is unused now (here and in `modules_test`).

Reply:
Good catch! Fixed.